src/Bridges/Bridges.jl

```diff
@@ -267,7 +281,11 @@ function runtests(
     # Load a non-bridged input model, and check that getters are the same.
     test = MOI.Utilities.UniversalFallback(MOI.Utilities.Model{eltype}())
     input_fn(test)
-    _test_structural_identical(test, model)
+    _test_structural_identical(
+        test,
+        model;
+        allow_constraint_function_error = allow_outer_constraint_function_error,
```

Comment:
```suggestion
        allow_constraint_function_error,
```
This should be sufficient instead of the long line?

Reply:
Note the `outer_`. I only want to allow it when comparing `test` with `model`, not with `inner`

Reply:
I'm open to better names ^^

Reply:
I, in fact, did not notice the `_outer_` 😆